src/consts/release.ts

```diff
@@ -0,0 +1,5 @@
+export enum ReleaseLabel {
+  APPLICATION = 'appstudio.openshift.io/application',
```

Comment:
I saw same const defined:
 src/consts/pipelinerun.ts:  APPLICATION = 'appstudio.openshift.io/application',

and the const has been enjoyed by other components. so i am considering how about we enjoy the existing const for this one.

Reply:
You can use the Application label from PipelineRunLabels in `src/consts/pipelinerun.ts`